pkg/volume/volume.go

```diff
@@ -103,3 +113,31 @@ func RenameDirectory(oldPath, newName string) (string, error) {
 	}
 	return newPath, nil
 }
+
+// ReadWriteModeFor returns the ReadWriteMode represented by the given boolean.
+// A value of true maps to ReadOnly, false to ReadWrite
+func ReadWriteModeFor(ro bool) ReadWriteMode {
```

Comment:
I find this sketchy as well. Is it possible to abstract this into a base class that volume builders will instantiate and use?